server/admin-function/set-admin.ts

```diff
@@ -0,0 +1,27 @@
+"use server";
+
+import { searchUserByEmail } from "@/server/admin-function/search-user-by-email";
+import { dbAdmin } from "@/lib/firebase-admin";
+
+export const setAdmin = async (
```

Comment:
I see that it's not used anywhere for now, the person that get the task to do set admin might not know you did it or forget, so go comment it in the code and mention that you created this logic.

Also, if you want to do a set admin, it might be good to return status to inform the admin that they updated somebody to admin.

ps: We probably won't do it this way, but finish this function and keep it

Reply:
great thinking, which logic are you mentioning?